jsonnet/telemeter/server/kubernetes.libsonnet

```diff
@@ -50,9 +48,13 @@ local whitelistFileName = 'whitelist';
       local rhdUsername = containerEnv.fromSecretRef('RHD_USERNAME', secretName, 'rhd.username');
       local rhdPassword = containerEnv.fromSecretRef('RHD_PASSWORD', secretName, 'rhd.password');
       local rhdClientID = containerEnv.fromSecretRef('RHD_CLIENT_ID', secretName, 'rhd.client_id');
-      local secretMount = containerVolumeMount.new(secretVolumeName, secretMountPath);
       local secretVolume = volume.fromSecret(secretVolumeName, secretName);
 
+      local whitelist = std.map(
+        function(rule) "--whitelist='%s'" % std.strReplace(rule, 'ALERTS', 'alerts'),
```

Comment:
does it make sense to do a std.asciiLower(str) here instead?

Reply:
no it does not make sense here; there may very well be metric names or labels in the whitelist that we do _not_ want completely lowercased. The only one we care about today is the `ALERTS` metric name.

Reply:
agreed